agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4216,6 +4219,38 @@ bool slave_thread::handle_client_association_request(Socket *sd, ieee1905_1::Cmd
     return send_cmdu_to_controller(cmdu_tx);
 }
 
+bool slave_thread::handle_1905_higher_layer_data_message(Socket *sd,
+                                                         ieee1905_1::CmduMessageRx &cmdu_rx)
+{
+    //only one backhaul manager (the slave) should return ACK for higher layer data message.
+    if (is_backhaul_manager) {
+        return true;
+    }
+
+    const auto mid = cmdu_rx.getMessageId();
+    LOG(DEBUG) << "Received HIGHER_LAYER_DATA_MESSAGE , mid=" << std::hex << int(mid);
+
+    auto tlvHigherLayerData = cmdu_rx.getClass<wfa_map::tlvHigherLayerData>();
+    if (!tlvHigherLayerData) {
+        LOG(ERROR) << "addClass wfa_map::tlvHigherLayerData failed";
+        return false;
+    }
+
+    const auto protocol       = tlvHigherLayerData->protocol();
+    const auto payload_length = tlvHigherLayerData->payload_length();
+    LOG(DEBUG) << "protocol: " << std::hex << int(protocol);
+    LOG(DEBUG) << "payload_length: " << std::hex << int(payload_length);
```

Comment:
```suggestion
    LOG(DEBUG) << "Protocol: " << std::hex << int(protocol);
    LOG(DEBUG) << "Payload-Length: " << std::hex << int(payload_length);
```
we should print the meaning of the data printed instead of the parameter name. this also makes future naming changes (that do not change the meaning) not required log print changes.

that being said:
1. please verify if the "payload_length" print is expected by some certification test before you change the print to "Payload-Length" and "Protocol"
2. if you change this and the certification tests don't use this print as reference (only our test-flows) then you must align our test-flows to the new print

Reply:
suggestion applied.